.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
+
 node_js:
-  - "0.10"
-  - "0.8"
+  - "0.12"
+  - "iojs"
```

Comment:
Node 4.0.0 was released today. I've updated the PR to reflect this change in the `.travis.yml` file.